test/api/searchResult.test.ts

```diff
@@ -71,54 +35,114 @@ describe('GET api/v1/search-results', () => {
             ...searchResult,
             id: faker.string.uuid(),
             userId: createdUser.id,
-            keyword: 'nothing',
+            keyword: 'litter',
           },
         ];
         await createManySearchResults(searchResultPayload);
```

Comment:
Let's add a lie rbeak to make it more redable:

```suggestion
        await createManySearchResults(searchResultPayload);
        
```